curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/6577333feab1e8e927014f03.md

```diff
@@ -0,0 +1,72 @@
+---
+id: 6577333feab1e8e927014f03
+title: Step 63
+challengeType: 20
+dashedName: step-63
+---
+
+# --description--
+
+Having `all([expression for i in iterable])`, means that a new list is created by evaluating `expression` for each `i` in `iterable`. After the `all()` function iterates over the newly created list, the list is deleted, since is no longer needed.
```

Comment:
```suggestion
Having `all([expression for i in iterable])`, means that a new list is created by evaluating `expression` for each `i` in `iterable`. After the `all()` function iterates over the newly created list, the list is deleted automatically, since it is no longer needed.
```